plugins/services/src/js/utils/__tests__/TaskTableUtil-test.js

```diff
@@ -9,13 +9,14 @@ describe('TaskTableUtil', function () {
   beforeEach(function () {
     this.getServiceHealth = MarathonStore.getServiceHealth;
     MarathonStore.getServiceHealth = function (prop) {
-      return this[prop].get('health');
+      return this[prop]['health'];
```

Comment:
Okay I didn't know that we do not use structs on tasks my bad. I just saw the `item.get` and concluded that it was a struct. My bad then forget what I said here. Nice catch!